src/api/utils.py

```diff
@@ -0,0 +1,32 @@
+import json
+import os.path
+
+
+def load_json(file_path):
+    """
+    Reads from JSON file and returns data.
+    If file not found, creates a new one.
+    """
+    if not os.path.exists(file_path):
+        with open(file_path, 'w') as wf:
+            json.dump({}, wf)
+
+    with open(file_path, 'r') as rf:
+        data = json.load(rf)
+    if not data:
```

Comment:
`{}` or `[]` are valid JSON data. They give the info that there is no data which might be valid case.

Reply:
Should it return an empty JSON, or a message `No professionals found`

Reply:
It is important to maintain abstractions in the codebase. That means you should not leak information between abstractions. _utils.py_ is a generic module that should usable in some other project as well. That too `load_json()` is just loading a JSON file. What that JSON contains or how it will be interpreted is upto the driver that calls this `utils.load_json()`. If you return "No professionals found", then you're leaking information in the sense that `utils.load_json()` should have no idea what a Professional is.

Reply:
For now, all the errors raised while opening the `db.json` file will be handled by the calling function of `load_json()` & `save_json` as the errors raised will be `500` error and this is only because a json file is our database, later when a database is incorporated we can generalise these errors into different HTTP exceptions. Do you this makes sense for now?